pyanaconda/payload/rpmostreepayload.py

```diff
@@ -368,15 +368,19 @@ def prepare_mount_targets(self, storage):
         # both mounts like /home (really /var/home) and %post scripts might
         # want to write to e.g. `/srv`, `/root`, `/usr/local`, etc. The
         # /var/lib/rpm symlink is also critical for having e.g. `rpm -qa` work
-        # in %post. We don't iterate *all* tmpfiles because we don't have the
-        # matching NSS configuration inside Anaconda, and we can't "chroot" to
-        # get it because that would require mounting the API filesystems in the
-        # target.
-        for varsubdir in ('home', 'roothome', 'lib/rpm', 'opt', 'srv',
-                          'usrlocal', 'mnt', 'media', 'spool', 'spool/mail'):
-            self._safe_exec_with_redirect("systemd-tmpfiles",
-                                          ["--create", "--boot", "--root=" + util.getSysroot(),
-                                           "--prefix=/var/" + varsubdir])
+        # in %post.
+        nsspaths = ("/etc/", "/usr/lib/")
+        for nfile in ("passwd", "group"):
+            infiles = [util.getSysroot() + path + nfile for path in nsspaths]
+            if all([os.path.isfile(f) for f in infiles]):
+                log.debug("copying over %s from sysroot", nfile)
+                with open("/etc/%s" % nfile, "w") as outf:
```

Comment:
Please change this to `.format()` instead of `%`.

Reply:
Done